### PR TITLE
Make the Flask logger configurable

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

File: cachito/web/config.py [hunk 1]
```diff
@@ -7,6 +7,11 @@
 
 class Config(object):
     """The base Cachito Flask configuration."""
+    # Additional loggers to set to the level defined in CACHITO_LOG_LEVEL
+    CACHITO_ADDITIONAL_LOGGERS = []
+    # This sets the level of the "flask.app" logger, which is accessed from current_app.logger
+    CACHITO_LOG_LEVEL = 'INFO'
```
Comment (lcarva) [inline cachito/web/config.py]: Can we set this to `DEBUG` for `DevelopmentConfig` and `TestingConfig` ?

File: cachito/web/app.py [hunk 2]
```diff
@@ -68,6 +70,17 @@ def create_app(config_obj=None):
     else:
         load_config(app)
 
+    # Configure logging
+    default_handler.setFormatter(
+        logging.Formatter(fmt=app.config['CACHITO_LOG_FORMAT'], datefmt='%Y-%m-%d %H:%M:%S')
+    )
+    app.logger.setLevel(app.config['CACHITO_LOG_LEVEL'])
+    for logger_name in app.config['CACHITO_ADDITIONAL_LOGGERS']:
+        logger = logging.getLogger(logger_name)
```
Comment (lcarva) [inline cachito/web/app.py]: Would this be for setting the logging level of third party libraries?
I think that's pretty neat.

Comment (mprahl) [reply cachito/web/app.py]: Yes, exactly!

File: cachito/web/config.py (hunk 1 above)
Comment (mprahl) [reply cachito/web/config.py]: Good point! I just added that.

**Comment by mprahl** [comment]
@lcarva could you please review again?

**Pull request merged**